packages/bootstrap-vue-next/src/plugins/toastController/index.ts

```
@@ -15,85 +33,193 @@ export const toastPlugin: Plugin = {
+          const toast = toasts.value.find((el) => el._self === _id)
+          if (!toast) return
+          toast.promise.stop?.()
+          if (toast.modelValue) {
+            await new Promise(async (resolve) => {
+              toast.modelValue = false
+              toast['onHidden'] = () => {
+                resolve(undefined)
+              }
+              await nextTick()
+              toast['onUpdate:modelValue']?.(false)
+            })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)